src/content/1.7/contribute/contribute-pull-requests/contribute_using_docker.md

```diff
@@ -55,10 +55,23 @@ prestashop-git | -- Installation successful! --
 prestashop-git | \n* Almost ! Starting web server now\n
 ```
 
-At this point, your PrestaShop installation is ready and the website is available at http://localhost:8001.
+At this point, your PrestaShop installation is ready and the website is available at http://localhost:8001. 
 
 The default credentials for the back-office are `demo@prestashop.com` / `prestashop_demo`.
 
+Default MySql credentials to connect using 3rd party programs like Sequel Pro and others:
+Username: `root`
```

Comment:
```suggestion
Username: `prestashop`
```

Reply:
> 

For me prestashop username not working. I'm using the latest repository.

Reply:
I'm using the latest version I'm able to connect with the `prestashop` user :thinking:

Reply:
I checked again, and you're right, the prestashop user isn't able to connect.